docs/configuration/backends/docker.md

```diff
@@ -269,80 +269,80 @@ services:
+|----------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `traefik.docker.network`                                             | Overrides the default docker network to use for connections to the container. [1]                                                                                                                                                |
+| `traefik.domain`                                                     | Sets the default base domain for the frontend rules. For more information, check the [Container Labels section's of the user guide "Let's Encrypt & Docker"](/user-guide/docker-and-lets-encrypt/#container-labels)              |
+| `traefik.enable=false`                                               | Disables this container in Traefik.                                                                                                                                                                                              |
+| `traefik.port=80`                                                    | Registers this port. Useful when the container exposes multiples ports.                                                                                                                                                          |
+| `traefik.tags=foo,bar,myTag`                                         | Adds Traefik tags to the Docker container/service to be used in [constraints](/configuration/commons/#constraints).                                                                                                              |
+| `traefik.protocol=https`                                             | Overrides the default `http` protocol                                                                                                                                                                                            |
+| `traefik.weight=10`                                                  | Assigns this weight to the container                                                                                                                                                                                             |
+| `traefik.backend=foo`                                                | Overrides the container name by `foo` in the generated name of the backend.                                                                                                                                                      |
```

Comment:
This is accurate for backends which are not segments. However, when we use segments, as explained in https://github.com/containous/traefik/issues/4284, the backend name is being set as `backend-<container-name>-backendsite1` where `traefik.backend=backendsite1` has been specified by
the user.